meta/no_language.yaml

```diff
@@ -4,12 +4,10 @@ rules:
     Please include a language field for your rule!
   languages: [yaml]
   patterns:
-  - pattern-either:
-    - pattern: |
-        rules: ...
+  - pattern-inside: 'rules: [..., $RULE, ...]'
+  - pattern: $RULE
   - pattern-not: |
-      rules:
-      - ...
-        languages: ...
-        ...
+      ...
+      languages: ...
+      ...
```

Comment:
We should remove the trailing ellipsis in these rules. It's just going to make them slower, and return more, unnecessary data.